configs/gunicorn.py

```diff
@@ -0,0 +1,37 @@
+from os import getenv
+from multiprocessing import cpu_count
+
+bind = f"0.0.0.0:{getenv('APP_PORT', '8000')}"
+
+# Log section
+if getenv('DEBUG') == "true":
```

Comment:
Worth a tweak to handle `True` and `true` in the config

```suggestion
if getenv('DEBUG').lower() == "true":
```